fe/pom.xml

```diff
@@ -608,7 +609,7 @@ under the License.
                             <goal>exec</goal>
                         </goals>
                         <configuration>
-                            <executable>${env.JAVA_HOME}/bin/java</executable>
+                            <executable>${java.home}/bin/java</executable>
```

Comment:
what does java.home mean? Why not use env.JAVA_HOME?

Reply:
![image](https://user-images.githubusercontent.com/9098473/72769348-5b3ed880-3c35-11ea-9e7d-f0f6368c985b.png)
maven document prefer to use java.home